DataLayer/Application/src/main/java/com/example/android/wearable/datalayer/MainActivity.kt

```diff
@@ -128,13 +129,39 @@ class MainActivity : ComponentActivity() {
             Uri.parse("wear://"),
             CapabilityClient.FILTER_REACHABLE
         )
+
+        if (isCameraSupported) {
+            lifecycleScope.launch {
+                try {
+                    capabilityClient.addLocalCapability(CAMERA_CAPABILITY).await()
```

Comment:
@codingjeremy I added the dynamic camera capability here per our earlier discussion, it wasn't too hard to add!